src/services/state.ts

```diff
@@ -276,6 +276,10 @@ export class StateService {
         this.state = GlobalConnectionState.Error;
         this.connectionBuildupState = connectionBuildupState;
         this.progress = 0;
-        this.unreadCount = 0;
+
+        // iOS devices are regularly disconnected, but this is normal behavior.
```

Comment:
Can you move this comment to the webclient service, where the `shouldResetUnreadCount` value is determined? Here it's a bit wrong, because the state service doesn't know anything about the task used.